2022.2.0 (#21435)
home-assistant/home-assistant.github.io: source/_posts/2022-02-02-release-20222.markdown

- [New Integrations](#new-integrations)
- [Integrations now available to set up from the UI](#integrations-now-available-to-set-up-from-the-ui)
- [If you need help...](#if-you-need-help)
- [Breaking Changes](#breaking-changes)
- [Farewell to the following](#farewell-to-the-following)
- [All changes](#all-changes)

## Updates from Nabu Casa

Before we dive into this release, we have some news and updates from
[Nabu Casa](https://www.nabucasa.com) to share with you.

### Annual subscriptions now available for the United States <!-- omit in toc -->

The ability to subscribe to [Home Assistant Cloud](https://www.nabucasa.com)
anually, was one of the most requested features since Nabu Casa was founded and has been
announced at the [State of the Open Home](/state-of-the-open-home/) in December.

As of today, annual subscriptions to Home Assistant Cloud are available
for the United States. 🎉

[Read more about the announcement on the Nabu Casa website](https://www.nabucasa.com/more-info/subscriptions-feb-22/)

### Improved infrastructure<!-- omit in toc -->

Nabu Casa's new payment system is part of revamped cloud system that they are

> rolling out, and this will bring more improvements the Home Assistant Cloud

Reviewer: ```suggestion
rolling out, and this will bring more improvements of the Home Assistant Cloud
```

services.

They already improved our remote access infrastructure to make sure it's routing

> as close to you as possible. Last week they rolle rolled out improvements

Reviewer: ```suggestion
as close to you as possible. Last week they rolled out improvements
```

to the infrastructure that significantly improved the speed of the
Google Assistant and Amazon Alexa integrations.

### Zack Barett and Mike Degatano join Nabu Casa <!-- omit in toc -->

We are also happy to announce [Zack Barett](https://github.com/zsarnett) and
[Mike Degatano](https://github.com/mdegat01) will be joining Nabu Casa to work
on Home Assistant.

Zack will be helping out on different aspects of Home Assistant, including the
[Home Assistant YouTube channel](https://www.youtube.com/channel/UCbX3YkedQunLt7EQAdVxh7w),
and the Home Assistant Frontend.

> Mike is going to be focussing on extending and improving the Home Assistant

Reviewer: ```suggestion
Mike is going to be focusing on extending and improving the Home Assistant
```

  table. Thanks, [@ludeeus]!
- Thanks to [@bramkragten], you can now select the period in the options of
  the statistics card.
- The Apple TV integration has been updated to support tvOS 15 and can now
  launch apps too! Thanks, [@postlund]!
- Thanks to [@bdraco], a device can now also display its hardware version in
  Home Assistant. The HomeKit, Xiaomi Miio, Bond, Roku, and WLED integrations
  have already adopted this.
- The Google Assistant integration can now be set up to use local
  fulfillment, thanks to [@LoekSangers]! Please note that this only applies to
  manual setups. This is already handled automatically when using the Home
  Assistant Cloud.
- ZHA devices now have identify buttons, so you know which light you are
  dealing with, thanks [@dmulcahey]! (Tip: also nice for visual notifications)
- We now enable Low Latency HLS (LL-HLS) by default to lower stream latency,
  thanks [@allenporter]!
- Magic Home (also known as FluxLED) had lots of improvements of love from [@bdraco]!
- [@Danielhiversen] added support for local access to Adax and Mill heaters. Nice!
- Tuya now supports Pet Feeders (thanks [@vauriga]), Circuit Breakers (thanks [@slydiman]),

>   and Alarms (thanks [@leeyuentuen]). Additionally, support for Fans had been improved.

Reviewer: ```suggestion
  and Alarms (thanks [@leeyuentuen]). Additionally, support for Fans has been improved.
```

- [August][august docs] (thanks [@bdraco])
- [Buienradar][buienradar docs] (thanks [@tedvdb])
- [EDL21][edl21 docs] (thanks [@StephanU])
- [GitHub][github docs] (thanks [@ludeeus])
- [Hunter Douglas PowerView][hunterdouglas_powerview docs] (thanks [@bdraco])
- [Kraken][kraken docs] (thanks [@eifinger])
- [Nexia/American Standard/Trane][nexia docs] (thanks [@bdraco])
- [Pentair ScreenLogic][screenlogic docs] (thanks [@bdraco])
- [Sensor.Community][luftdaten docs] (thanks [@frenck])
- [SiteSage Emonitor][emonitor docs] (thanks [@bdraco])
- [Tesla Powerwall][powerwall docs] (thanks [@bdraco])

And the following integrations now have entity categories:

- [Aurora ABB PowerOne Solar PV][aurora_abb_powerone docs] (thanks [@davet2001])
- [DSMR][dsmr docs] (thanks [@starkillerOG])
- [GitHub][github docs] (thanks [@ludeeus])
- [Hyperion][hyperion docs] (thanks [@starkillerOG])
- [Kostal Plenticore Solar Inverter][kostal_plenticore docs] (thanks [@stegm])
- [Mazda Connected Services][mazda docs] (thanks [@bdr99])
- [Motion Blinds][motion_blinds docs] (thanks [@starkillerOG])
- [NETGEAR][netgear docs] (thanks [@starkillerOG])

## New Integrations

> We welcome the following new integration this release:

Reviewer: ```suggestion
We welcome the following new integrations this release:
```

- [Launch Library][launch_library docs], done by [@DurgNomis-drol]
- [LG webOS Smart TV][webostv docs], done by [@thecode]
- [PVOutput][pvoutput docs], done by [@frenck]
- [Sensibo][sensibo docs], done by [@gjohansson-ST]
- [SolaX Power][solax docs], done by [@tokenize47]
- [Vallox][vallox docs], done by [@slovdahl]
- [Version][version docs], done by [@ludeeus]
- [Whois][whois docs], done by [@frenck]

## If you need help...

...don't hesitate to use our very active [forums](https://community.home-assistant.io/) or join us for a little [chat](https://discord.gg/c5DvZ4e).

Experiencing issues introduced by this release? Please report them in our [issue tracker](https://github.com/home-assistant/core/issues). Make sure to fill in all fields of the issue template.

<!--more-->

## Breaking Changes

Below is a listing of the breaking change for this release, per subject or
integration. Click on one of those to read more about the breaking change
for that specific item.

{% details "Energy Dashboard" %}

The configuration of the Energy Dashboard has moved.

Previously this was found as a menu item in the configuration screen,
but has now been moved into the "Dashboard" menu item.

> This keeps the configuration of all dashboard in a single place.

Reviewer: ```suggestion
This keeps the configuration of all dashboards in a single place.
```

([@zsarnett] - [#11386])

[@zsarnett]: https://github.com/zsarnett
[#11386]: https://github.com/home-assistant/frontend/pull/11386

{% enddetails %}

{% details "Python 3.8 support dropped" %}

Python 3.8 support has previously been deprecated and now has been removed.
Home Assistant now requires Python 3.9 to run.

> If you are using Home Assistant OS, Home Assistant Container, or Home Assistant Supervised installation method, you don't have to do anything. In those cases, we all handle this for you. Your system has been using Python 3.9 for quite some time already.

Reviewer: ```suggestion
If you are using Home Assistant OS, Home Assistant Container, or Home Assistant Supervised installation method, you don't have to do anything. In those cases, we handle this all for you. Your system has been using Python 3.9 for quite some time already.
```

([@frenck] - [#63883])

{% enddetails %}

{% details "GPIO Integrations" %}

> As of this release, all integration intefacing with GPIO directly, have been

Reviewer: ```suggestion
As of this release, all integrations interfacing with GPIO directly, have been
```

deprecated.

There are multiple reasons for this, which includes a general low usage of

> these integrations. For most Home Assistant installation, GPIO isn't easily

Reviewer: ```suggestion
these integrations. For most Home Assistant installations, GPIO isn't easily
```

> usable and more often the integrations are unmainted.

Reviewer: ```suggestion
usable and more often the integrations are unmaintained.
```

More information can be found in

> [Architectural Decission Record 0019](https://github.com/home-assistant/architecture/blob/master/adr/0019-GPIO.md).

Reviewer: ```suggestion
[Architectural Decision Record 0019](https://github.com/home-assistant/architecture/blob/master/adr/0019-GPIO.md).
```

to a remote owfs server. The use of the SysBus is deprecated, however,
connecting to a remote owfs server is not.

{% enddetails %}

{% details "Binary Sensors" %}

Previously, `binary_sensor` entities could have the state `on`, `off`, and
in case the device was unreachable: `unavailable`.

However, compared to other entities, it wasn't able to have the `unknown` state,
which has been added this release.

So, a `binary_sensor` entity can now have the `on`, `off`, `unavailable`, or `unknown` state.

You might need to adapt your automations or scripts to take this new
`unknown` state into account.

([@frenck] - [#60193]) ([binary_sensor docs])

{% enddetails %}

{% details "Groups: Locks" %}

The behavior of grouped locks has been reversed such that "unlocked" is
mapped to `on` and "locked" to off.

This means that:

- A group with mixed domains will be considered `on` if any lock included in
  the group is unlocked.
- A group of locks will be considered "unlocked" if any lock included in the
- group is unlocked.

([@emontnemery] - [#62580]) ([lock docs]) ([group docs])

{% enddetails %}

{% details "Fan/Humidifer/Light/Remote/Siren/Switch/Vacuum" %}

> Previously, toggle-based entities (like `fan`, `light`, `switch`, `remote`, `siren`, `vacuum`, `humidifier`) could have the state `on` or `off`, and in case the device was unreachable: `unavailable`. However, compared to other entities, toggle entities aren't able to have an `unknown` state, this now has changed.

Reviewer: ```suggestion
Previously, toggle-based entities (like `fan`, `light`, `switch`, `remote`, `siren`, `vacuum`, `humidifier`) could have the state `on` or `off`, and in case the device was unreachable: `unavailable`. However, compared to other entities, toggle entities weren't able to have an `unknown` state, this now has changed.
```

> As of now, toggle-based entities can now have the `on`, `off`, `unavailable`, or `unknown` state.

Reviewer: ```suggestion
As of now, toggle-based entities can have the `on`, `off`, `unavailable`, or `unknown` state.
```

You might need to adapt your automations or scripts to take this new `unknown` state into account.

([@frenck] - [#64621])

{% enddetails %}

{% details "Advantage Air" %}

> The `advantage_air.set_myzone` service has been depreciated and will be removed

Reviewer: ```suggestion
The `advantage_air.set_myzone` service has been deprecated and will be removed
```

in a future version of Home Assistant.

Please use the `select.select_option` service on the MyZone select entity to change this value on supported systems instead.

([@Bre77] - [#58777]) ([advantage_air docs])

{% enddetails %}

{% details "Android TV" %}

The Android TV integration migrated to configuration via the
UI. Configuring Android TV via YAML configuration has been
deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration after upgrading.

- Add config_flow to AndroidTV integration ([@ollo69] - [#54444]) ([androidtv docs]) (breaking-change)

{% enddetails %}

{% details "API" %}

The `/api/discovery_info` endpoint was deprecated in 2021.10, and has since only
returned blank values. From this version, the `/api/discovery_info` endpoint has

> been removed completly.

Reviewer: ```suggestion
been removed completely.
```

{% enddetails %}

{% details "Apple Push Notification Service (APNS)" %}

The Apple Push Notification Service (APNS) integration has been deprecated and
will be removed in Home Assistant 2022.4. Please note, this does not affect
our mobile companion apps; as they do not use this integration for their
notifications.

([@frenck] - [#64629]) ([apns docs])

{% enddetails %}

{% details "Aurora ABB PowerOne Solar PV" %}

The previously deprecated YAML configuration of the Aurora ABB PowerOne Solar PV
integration has been removed.

Aurora ABB PowerOne Solar PV is now configured via the UI, any existing YAML
configuration has been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@tkdrob] - [#62317]) ([aurora_abb_powerone docs])

{% enddetails %}

{% details "AVM FRITZ!Box Tools" %}

The previously deprecated YAML configuration of the AVM FRITZ!Box Tools
integration has been removed.

AVM FRITZ!Box Tools is now configured via the UI, any existing YAML
configurationhas been imported in previous releases and can now be safely
removed from your YAML configuration files.

[@epenet] - [#63857]) ([fritz docs])

----

> The `reboot` and `reconnect` services are deprecated and will be removed in future release. Please use the newly provided button entities instead.

Reviewer: ```suggestion
The `reboot` and `reconnect` services are deprecated and will be removed in a future release. Please use the newly provided button entities instead.
```

integration has been removed.

Efergy is now configured via the UI, any existing YAML configuration
has been imported in previous releases and can now be safely removed from your
YAML configuration files.

([@tkdrob] - [#61520]) ([efergy docs])

{% enddetails %}

{% details "Enphase Envoy" %}

The previously deprecated YAML configuration of the Enphase Envoy integration
has been removed.

Enphase Envoy is now configured via the UI, any existing YAML configuration
has been imported in previous releases and can now be safely removed from your
YAML configuration files.

([@tkdrob] - [#61840]) ([enphase_envoy docs])

{% enddetails %}

{% details "Environment Canada" %}

The previously deprecated YAML configuration of the Environment Canada
integration has been removed.

Environment Canada is now configured via the UI, any existing YAML configuration
has been imported in previous releases and can now be safely removed from your
YAML configuration files.

([@tkdrob] - [#61839]) ([environment_canada docs])

---

The sensors no longer have `Timestamp` as an extra state attribute, instead,
use the new `timestamp` sensor that can be used in automations that track state changes.

> The sensors alert sensors (`advisories`, `endings`, `statements`, `warnings`, and `watches`) state is now a count of the number of active alerts for the sensor type

Reviewer: ```suggestion
The alert sensors (`advisories`, `endings`, `statements`, `warnings`, and `watches`) state is now a count of the number of active alerts for the sensor type
```

and the extra state attributes contain the alert text and alert timestamp.

The attributes are named `alert_<x>` and `alert_time_<x>` where <x> is 1 for alert 1, 2 for alert 2, etc.

([@gwww] - [#58615]) ([environment_canada docs])

{% enddetails %}

{% details "Fan" %}

This is more a breaking change for custom integration developers.

Now that all core fans are using the new fan entity model, the compatibility shim
has been removed. Custom integrations that have not yet switched to the new

> fan model. For more information see:

Reviewer: ```suggestion
fan model may be affected. For more information see:
```

that to use the new energy sensors instead.

([@rappenze] - [#63697]) ([fibaro docs])

{% enddetails %}

{% details "Flume" %}

The previously deprecated YAML configuration of the Flume integration has been removed.

Flume is now configured via the UI, any existing YAML configuration has been imported
in previous releases and can now be safely removed from your YAML configuration files.

([@tkdrob] - [#61517]) ([flume docs])

{% enddetails %}

{% details "FortiOS" %}

The integration now requires FortiOS 6.4.3 and newer.

([@kimfrellsen] - [#61970]) ([fortios docs])

{% enddetails %}

{% details "Foscam" %}

The previously deprecated YAML configuration of the Foscam integration
has been removed.

Foscam is now configured via the UI, any existing YAML configuration
has been imported in previous releases and can now be safely removed from your
YAML configuration files.

([@tkdrob] - [#61761]) ([foscam docs])

{% enddetails %}

{% details "Foursquare" %}

The data associated with `foursquare.checkin` event is now available as a
dictionary item with key "text".

If you use this event in your automation or scripts, you'd need to adjust to

> to match this change.

Reviewer: ```suggestion
match this change.
```

The previously deprecated YAML configuration of the Honeywell Total Connect Comfort
integration has been removed.

Honeywell Total Connect Comfort is now configured via the UI, any existing YAML
configurationhas been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@tkdrob] - [#62469]) ([honeywell docs])

{% enddetails %}

{% details "iOS" %}

The Home Assistant iOS/macOS app supports notification actions defined [in the notification itself](https://companion.home-assistant.io/docs/notifications/actionable-notifications/); the old method of defining push categories in the iOS integration is now deprecated and will be removed in the future.

([@zacwest] - [#61078]) ([ios docs])

{% enddetails %}

{% details "Keenetic NDMS2 Router" %}

The previously deprecated YAML configuration of the Keenetic NDMS2 Router
integration has been removed.

Keenetic NDMS2 Router is now configured via the UI, any existing YAML
configurationhas been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@epenet] - [#63860]) ([keenetic_ndms2 docs])

{% enddetails %}

{% details "Launch Library" %}

> The Launch Libraryintegration migrated to configuration via the

Reviewer: ```suggestion
The Launch Library integration migrated to configuration via the
```

UI. Configuring Launch Library via YAML configuration has been
deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration after upgrading.

([@DurgNomis-drol] - [#62416]) ([launch_library docs])

---

Attributes on the current sensor is changed. Names of these attributes are
changed to better describe what information they provide. (Ex. `agency` changed
name to `provider` as this is the more correct terminology.) New attributes are
added to give more information about where it takes place.

> The `stream` attribute ios moved to the new sensor, and changed it's name to `stream_live` as it represent if the stream is live and not if a stream is

Reviewer: ```suggestion
The `stream` attribute ios moved to the new sensor, and changed its name to `stream_live` as it represents if the stream is live and not if a stream is
```

available.

> And finally, the attribute `launch_time` moved into it's own sensor. This made

Reviewer: ```suggestion
And finally, the attribute `launch_time` moved into its own sensor. This made
```

it possible to add some information about the launch window start and end as attributes for the new sensor. This sensor will also contain the `stream_live` attribute.

([@DurgNomis-drol] - [#64203]) ([launch_library docs])

{% enddetails %}

{% details "LG webOS Smart TV" %}

The LG webOS Smart TV integration migrated to configuration via the
UI. Configuring LG webOS Smart TV via YAML configuration has been
deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration after upgrading.

The [turn on action](/integrations/webostv/#turn-on-action) for turning on the

> TV via `WakeOnLan` or `HDMI-CEC` was using a service call via YAML, the turn on action is changed to use a custom automation trigger `webostv.turn_on` or a device trigger automation that can be setup from the UI.

Reviewer: ```suggestion
TV via `WakeOnLan` or `HDMI-CEC` was using a service call via YAML, and the turn on action is now changed to use a custom automation trigger `webostv.turn_on` or a device trigger automation that can be setup from the UI.
```

**Example trigger automation:**

```yaml
automation:
  trigger:
    - platform: webostv.turn_on
      entity_id: media_player.lg_webos_smart_tv
  action:
    - service: wake_on_lan.send_magic_packet
      data:
        mac: AA-BB-CC-DD-EE-FF
```

([@thecode] - [#64117]) ([webostv docs])

{% enddetails %}

{% details "Logitech Squeezebox" %}

The previously deprecated YAML configuration of the Logitech Squeezebox
integration has been removed.

Logitech Squeezebox is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

NETGEAR is now configured via the UI, any existing YAML
configurationhas been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@epenet] - [#63862]) ([netgear docs])

{% enddetails %}

{% details "Nissan Leaf" %}

The service to start a charge has been deprecated; it is replaced with a button.

Replace the `nissan_leaf.start_charge` service in any existing automations with `button.press` for the new `button.start_NICKNAME_charging` entity.

The `nissan_leaf.start_charge` service is now deprecated and will be removed
in a future release.

([@filcole] - [#62948]) ([nissan_leaf docs])

{% enddetails %}

{% details "Nmap Tracker" %}

The previously deprecated YAML configuration of the Nmap Tracker
integration has been removed.

Nmap Tracker is now configured via the UI, any existing YAML
configurationhas been imported in previous releases and can now be safely
removed from your YAML configuration files.

([@epenet] - [#63863]) ([nmap_tracker docs])

{% enddetails %}

{% details "Nuki" %}

The previously deprecated YAML configuration of the Nuki
integration has been removed.

Nuki is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

removed from your YAML configuration files.

([@tkdrob] - [#62470]) ([nuki docs])

{% enddetails %}

{% details "OpenGarage" %}

The previously deprecated YAML configuration of the OpenGarage
integration has been removed.

OpenGarage is now configured via the UI, any existing YAML configuration
has been imported in previous releases and can now be safely removed from your
YAML configuration files.

- Remove deprecated yaml config from opengarage ([@tkdrob] - [#61961]) ([opengarage docs]) (breaking-change)

{% enddetails %}

{% details "Philips TV" %}

The previously deprecated YAML configuration of the Philips TV
integration has been removed.

Philips TV is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

removed from your YAML configuration files.

([@tkdrob] - [#62471]) ([philips_js docs])

{% enddetails %}

{% details "PVOutput" %}

The PVOutput integration migrated to configuration via the
UI. Configuring PVOutput via YAML configuration has been
deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration after upgrading.

Additionally, all sensor attributes of the PVOutput sensor,
have been extracted in their own dedicated sensors.

This applies to the following PVOutput attributes:

- `energy_generation`
- `power_generation`
- `energy_consumption`
- `power_consumption`
- `efficiency`
- `temperature`
- `voltage`

> If you relied on these in your automation or scripts, you need to adopt those to use the new separate sensors instead. The attributes are currently available, however, deprecated and will be removed in Home Assistant Core 2022.4.

Reviewer: ```suggestion
If you relied on these in your automation or scripts, you need to adapt those to use the new separate sensors instead. The attributes are currently available, however, deprecated and will be removed in Home Assistant Core 2022.4.
```

([@frenck] - [#62667] [#62894]) ([pvoutput docs])

{% enddetails %}

{% details "ReCollect Waste" %}

> ReCollect Waste no longer stories "next pickup" information as attributes of the "current pickup" sensor; instead, both pickups are now their own sensors.

Reviewer: ```suggestion
ReCollect Waste no longer stores "next pickup" information as attributes of the "current pickup" sensor; instead, both pickups are now their own sensors.
```

([@bachya] - [#62558]) ([recollect_waste docs])

{% enddetails %}

{% details "RIVM Stookalert" %}

The previously deprecated YAML configuration of the Stookalert
integration has been removed.

Stookalert is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

removed from your YAML configuration files.

([@frenck] - [#61007]) ([stookalert docs])

{% enddetails %}

{% details "Samsung SyncThru Printer" %}

The previously deprecated YAML configuration of the Samsung SyncThru Printer
integration has been removed.

Samsung SyncThru Printer is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

([@tkdrob] - [#62541]) ([syncthru docs])

{% enddetails %}

{% details "Sensibo" %}

The Sensibo integration migrated to configuration via the
UI. Configuring Sensibo via YAML configuration has been
deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration after upgrading.

([@gjohansson-ST] - [#60900]) ([sensibo docs])

{% enddetails %}

{% details "Sensor.Community (previously: Luftdaten)" %}

The Luftdaten integration has been renamed to Sensor.Community; which is
the new name of the Luftdaten project.

The previously deprecated YAML configuration of this integration
has been removed.

Sensor.Community is now configured via the UI, any existing YAML configuration
has been imported in previous releases and can now be safely removed from your
YAML configuration files.

([@frenck] - [#61748] [#62865]) ([luftdaten docs])

{% enddetails %}

{% details "SMA Solar" %}

The previously deprecated YAML configuration of the SMA Solar
integration has been removed.

SMA Solar is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

{% enddetails %}

{% details "SwitchBot" %}

The SwitchBot integration migrated to configuration via the
UI in a previous release already. Configuring SwitchBot via YAML configuration
has now been deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is already automatically imported; and thus
can be safely removed from your YAML configuration.

([@tkdrob] - [#62583]) ([switchbot docs])

{% enddetails %}

{% details "Synology DSM" %}

The `reboot` and `shutdown` services are deprecated and will be removed in future release. Please use the new button entities instead.

([@mib1185] - [#57352]) ([synology_dsm docs]) (breaking-change)

{% enddetails %}

{% details "Templates: Binary Sensors" %}

Template trigger binary sensor should start with `unknown` value rather than always starting with `false`/`off`. This matches the template binary sensor behavior
with other sensors.

([@amosyuen] - [#62769]) ([template docs])

{% enddetails %}

{% details "TP-Link Kasa Smart" %}

The previously deprecated YAML configuration of the TP-Link Kasa Smart
integration has been removed.

TP-Link Kasa Smart is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

removed from your YAML configuration files.

([@bdraco] - [#62457]) ([tplink docs])

{% enddetails %}

{% details "Trafikverket Train" %}

> The planned time, estimated time and and actual time sensor attributes have

Reviewer: ```suggestion
The planned time, estimated time and actual time sensor attributes have
```

> been correct to be UTC isoformatted string which is standard in Home Assistant.

Reviewer: ```suggestion
been corrected to be UTC isoformatted strings which is standard in Home Assistant.
```

([@decompil3d] - [#61837]) ([vesync docs])

{% enddetails %}

{% details "Viessmann ViCare" %}

The scan interval option has been removed, since the default should be fine
in almost all cases.

If you really need more fine-grained control you can disable automatic updates
in the system settings for the Viessmann ViCare integration and automate the
polling of the entities with the
[`homeassistant.update_entity`](/docs/scripts/service-calls#homeassistant-services)
service.

([@oischinger] - [#63343]) ([vicare docs])

{% enddetails %}

{% details "VIZIO SmartCast" %}

If an integrated `vizio` TV supports apps and the source is currently a non-app
input (e.g. HDMI-1), the `app_id` and `app_name` attributes will no longer be in the media player entity's state.

When the TV is using an app, the `app_name` will still always be visible and the `app_id` will continue to only be shown if the app is not recognized.

([@raman325] - [#64025]) ([vizio docs])

{% enddetails %}

{% details "VLC media player Telnet" %}

The previously deprecated YAML configuration of the VLC media player Telnet
integration has been removed.

VLC media player Telnet is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

removed from your YAML configuration files.

([@tkdrob] - [#62542]) ([vlc_telnet docs])

{% enddetails %}

{% details "Waze Travel Time" %}

The previously deprecated YAML configuration of the Waze Travel Time
integration has been removed.

Waze Travel Time is now configured via the UI, any existing YAML

> configurationhas been imported in previous releases and can now be safely

Reviewer: ```suggestion
configuration has been imported in previous releases and can now be safely
```

removed from your YAML configuration files.

([@tkdrob] - [#62543]) ([waze_travel_time docs])

{% enddetails %}

{% details "WebSocket API / Media Source" %}

This concerns custom integration developers only.

The parameters for `async_sign_path` have changed. Refresh token is now a keyword parameter. If called as part of a HTTP or WebSocket request, the users refresh token will automatically be used.

([@balloob] - [#64337])

{% enddetails %}

{% details "Whois" %}

The Whois integration migrated to configuration via the
UI. Configuring Whois via YAML configuration has been
deprecated and will be removed in a future Home Assistant release.

Your existing YAML configuration is automatically imported on upgrade to this
release; and thus can be safely removed from your YAML configuration after upgrading.

([@frenck] - [#63069]) ([whois docs])

{% enddetails %}

{% details "Z-Wave JS" %}

Due to a schema change, `zwave-js-server` version 1.14.0 or greater is required.
Ensure you are using the latest version of `zwave-js-server` (or the add-ons

> that provides those).

Reviewer: ```suggestion
that provide those).
```

Currently zeroconf matching only allows matching the `macaddress`, `model`, and `manufacturer` properties along with the `name` from the ZeroconfServiceInfo.

Since properties are arbitrarily defined by the zeroconf service,
the list of named properties has grown over time.

Matching now allows for any arbitrarily defined property.
All property matches must be lowercase, wildcards are supported

The top level keys `model`, `manufacturer`, and `macaddress`
are now deprecated from the `manifest.json` file and should
be moved into a `properties` dict.

For example:
```diff
-    {"type":"_airplay._tcp.local.","model":"appletv*"}
+    {"type":"_airplay._tcp.local.","properties":{"model":"appletv*"}}
```

([@bdraco] - [#62133]) ([zeroconf docs])

{% enddetails %}

## Farewell to the following

The following integrations are no longer available as of this release:

- **TaHoma** has been removed. The integration has been broken and now been
  superseded by the [Overkiz](/integrations/overkiz) integration.
  ([@iMicknl] - [#62607])
- **Arduino** the now long time deprecated integration has been removed.
  ([@frenck] - [#63406])

> - **EE Bright Box** has been broken en disabled for over a year now.

Reviewer: ```suggestion
- **EE Bright Box** has been broken and disabled for over a year now.
```

Author: Lol, that just Dutch 🤣
